### PR TITLE
feat(tier4_planning_msgs): add acceleration field in VelocityLimitConstraints.msg

### PR DESCRIPTION
**Pull request by go-sakayori**
## Related Links
https://github.com/orgs/autowarefoundation/discussions/5209

## Description
This PR adds an acceleration field in the `VelocityLimitConstraints.msg`, enabling to consider acceleration in packages such as external velocity limiter etc.

## Remarks
No regression as checked in [TIER IV internal evaluator](https://evaluation.tier4.jp/evaluation/reports/aefeb712-5add-595e-ac8f-33d68f6ea911?project_id=prd_jt)

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [x] Code is properly formatted
- [x] Assign PR to reviewer

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [ ] Commits are properly organized and messages are according to the guideline
- [ ] Code is properly formatted
- [ ] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets
- [ ] Write [release notes][release-notes]

## CI Checks

- **Build and test for PR**: Required to pass before the merge.
- **Check spelling**: NOT required to pass before the merge. It is up to the reviewer(s). See [here][spell-check-dict] if you want to add some words to the spell check dictionary.

[release-notes]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/563774416
[spell-check-dict]: https://github.com/tier4/autoware-spell-check-dict#how-to-contribute

### CODE REVIEW

**Review by yukkysaito — APPROVED** [review]
LGTM

**Comment by go-sakayori** [comment]
@yukkysaito 
I don't have the authority to merge, so can you? Thanks in advance.

File: tier4_planning_msgs/msg/VelocityLimitConstraints.msg [hunk 1]
```diff
@@ -1,3 +1,4 @@
+float32 max_acceleration
```
Comment (yukkysaito) [inline tier4_planning_msgs/msg/VelocityLimitConstraints.msg:2]: If possible, could you please add a comment here indicating that this is the maximum signed acceleration?

Comment (go-sakayori) [reply tier4_planning_msgs/msg/VelocityLimitConstraints.msg:2]: Fixed in 1377eef427a564e0e8e3fe07ed17a749b1c58ad9

**Pull request merged**